src/store.js

```diff
@@ -0,0 +1,109 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import { auth, database } from '@/plugins/firebase'
+import createPersistedState from 'vuex-persistedstate'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    user: {
+      name: null,
+      email: null,
+      uid: null
+    },
+    uid: null
+  },
+  mutations: {
+    saveUID (state, uid) {
+      state.uid = uid
+    },
+    saveUSER (state, user) {
+      state.user = user
+    }
+  },
+  actions: {
+    signManifest ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        let eventKey = database.ref('/firms/private').push().key
+        let dateCalc
+        let dateObj = new Date()
+        let month = dateObj.getMonth() + 1
+        let day = dateObj.getDate()
+        if (day <= 9) {
+          day = `0${day}`
+        }
+        if (month <= 9) {
+          month = `0${month}`
+        }
+        let year = dateObj.getFullYear()
+        dateCalc = `${year}/${month}/${day}`
```

Comment:
You might be interested in [the moment.js library](https://momentjs.com) to make date calculations and rendering easier 🙂

Reply:
I've been thinking about it but, really, it's the only place where I make use of a date, and importing a whole library seemed like too much to me.